presto-orc/src/main/java/com/facebook/presto/orc/metadata/OrcMetadataReader.java

```diff
@@ -391,6 +393,21 @@ private static DateStatistics toDateStatistics(HiveWriterVersion hiveWriterVersi
                 dateStatistics.hasMaximum() ? dateStatistics.getMaximum() : null);
     }
 
+    private static TimestampStatistics toTimestampStatistics(HiveWriterVersion hiveWriterVersion, OrcProto.TimestampStatistics timestampStatistics, boolean isRowGroup)
+    {
+        if (hiveWriterVersion == ORIGINAL && !isRowGroup) {
```

Comment:
Sorry I think we could include the file and stripe level statistics.@Dain Should we have to check the ORC Writer version for Timestamp statistics similar to Date and String statistics.(https://issues.apache.org/jira/browse/HIVE-8732)